src/MPI_for_CUDA_backend/send_recv_buff.cpp

```diff
@@ -47,8 +49,8 @@ int main(int argc, char *argv[]) {
 
   {
     /* -------------------------------------------------------------------------------------------
-      Create a SYCL buffer in each rank. The sycl::buffer created in each rank will
-      manage the copy of data to and from the device as required.
+      Create a SYCL buffer in each rank. The sycl::buffer created in each rank
+    will manage the copy of data to and from the device as required.
```

Comment:
```suggestion
      will manage the copy of data to and from the device as required.
```

Reply:
Done + fixed comment indentation across the file and cropped the comment separators (`/*-----...`) at col 80